packages/kit/src/exports/vite/utils.js

```diff
@@ -19,7 +19,14 @@ export async function get_vite_config(config, config_env) {
 	if (!loaded) {
 		throw new Error('Could not load Vite config');
 	}
-	return { ...loaded.config, mode: config_env.mode };
+	return {
+		...loaded.config,
+		// CLI opts
+		mode: config_env.mode,
+		logLevel: config.logLevel,
+		clearScreen: config.clearScreen,
+		optimizeDeps: { force: config.optimizeDeps.force }
```

Comment:
is this enough or do you need to clone the whole optimizeDeps config? right now it is disabled in build by default but in a future version or user config it may be ebabled?

Reply:
ahh, that's only the cli values and merge happen s later

Reply:
It's only `--force` that you can specify via the CLI. I copied this from the Vite codebase